spec/search_builders/publication_search_builder_spec.rb

```diff
@@ -16,6 +16,7 @@
       it 'filters for id and type' do
         expect(solr_params[:fq]).to eq ['authors_ssim:"123abc"', 'type_ssi:Publication']
         expect(solr_params[:rows]).to eq 1000
+        expect(solr_params[:sort]).to eq 'created_year_isim desc'
```

Comment:
Sorting by a multivalued field works now?  That's been a troublespot in the past.

Reply:
It appears to ...
![screen shot 2018-11-09 at 1 48 51 pm](https://user-images.githubusercontent.com/588335/48281964-532abc80-e426-11e8-9f81-d311421caa4d.png)